cli/src/main/scala/com/advancedtelematic/tuf/cli/Cli.scala

```diff
@@ -101,6 +102,16 @@ object Cli extends App with VersionInfo {
       .text("Path where this executable will look for keys, by default it's the `user-keys` directory in `home-dir`")
   }
 
+  lazy val expirationOpts: OptionParser[Config] => Seq[OptionDef[_, Config]] = { parser =>
+    Seq(
+      parser.opt[Instant]("expires")
+        .text("UTC instant such as '2018-01-01T00:01:00Z'")
+        .toConfigOptionParam('expireOn),
+      parser.opt[Period]("expire-after")
+        .text("Expiration delay in years, months and days (each optional, but in that order), such as '1Y3M5D'")
+        .toConfigOptionParam('expireAfter))
+  }
```

Comment:
I wanted to be backwards-compatible. What I didn't do is this though: "Additionally, the sign command should be changed so that if the unsigned role contains an expire date that is later than what the default date would be, the unsigned role expire date should be used."

Reply:
I think we don't need to be backwards compatible, when we bump the garage-sign version on meta-updater we could also add this param. @patrickvacek  ?

Yeah, I was thinking requiring this would be a way around that..

Reply:
Sounds fine to me as long as we don't forget to do it!

Reply:
I guess then it would make more sense to work the other way around: use the date in the unsigned role by default, error when signing an expired role (unless `--force` is given), use the date given on the command line if there is one.

Reply:
yes that sounds reasonable...

Reply:
Ok, will do that.

Reply:
Added 2nd commit to do that. (_Note to self_: squash before merging.)